build-config/spotbugs-exclude.xml

```diff
@@ -21,11 +21,15 @@
         <Bug pattern="NP_LOAD_OF_KNOWN_NULL_VALUE"/>
     </Match>
     <Match>
-        <class name="org.ballerinalang.sql.utils.QueryUtils"/>
+        <class name="org.ballerinalang.sql.nativeimpl.QueryProcessor"/>
         <Bug pattern="SQL_PREPARED_STATEMENT_GENERATED_FROM_NONCONSTANT_STRING"/>
     </Match>
     <Match>
-        <class name="org.ballerinalang.sql.utils.ExecuteUtils"/>
+        <class name="org.ballerinalang.sql.nativeimpl.QueryProcessor"/>
+        <Bug pattern="OBL_UNSATISFIED_OBLIGATION"/>
```

Comment:
This is valid bug. This is shown if the IO streams are not closed properly. Was this followed up?

Reply:
Resolved with 0359f47